solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -1,84 +1,102 @@
-   7. S3_EXTERNAL_HTTP_NODEPORT - If you want to use different port for HTTP Port to IO service, then export this variable with another port (Default value S3_EXTERNAL_HTTP_NODEPORT="30080").
-   8. S3_EXTERNAL_HTTPS_NODEPORT - If you want to use different port for HTTPS Port to IO service, then export this variable with another port (Default value S3_EXTERNAL_HTTPS_NODEPORT="30443").
-   9. SOLUTION_CONFIG_TYPE - There are two types config for solution yaml file manual and automated. Automated means solution.yaml is created automatically but it needs VM with required specifications and manual means we need to create solution.yaml file and replace it at script location. If you wanted to create solution yaml manually, then create it and place it at script location and configure SOLUTION_CONFIG_TYPE variable as manual (Default value SOLUTION_CONFIG_TYPE=manual).
+Note:
+   1. CORTX_SCRIPTS_BRANCH : If you want to use another cortx-K8s branch then export this variable with your branch (Default value CORTX_SCRIPTS_BRANCH="v0.2.1").
+   2. CORTX_SCRIPTS_REPO : If you want to use another cortx-K8s repo (like your fork), export this variable with your repo (Default value CORTX_SCRIPTS_REPO="Seagate/cortx-k8s").
+   3. CORTX_ALL_IMAGE : In automated case we are using latest cortx-all image. If you want to use different image then export the image by this variable (Default value CORTX_ALL_IMAGE=ghcr.io/seagate/cortx-all:2.0.0-latest).
+   4. CORTX_SERVER_IMAGE : Also, if you want to use different server image then export this variable with new image (Default value CORTX_SERVER_IMAGE=ghcr.io/seagate/cortx-rgw:2.0.0-latest).
+   5. CORTX_DATA_IMAGE : Also, if you want to use different data image then export this variable with new image (Default value CORTX_DATA_IMAGE=ghcr.io/seagate/cortx-data:2.0.0-latest).
+   6. CONTROL_EXTERNAL_NODEPORT : If you want to use different port for control service, export this variable with another port (Default value CONTROL_EXTERNAL_NODEPORT="31169").
+   7. S3_EXTERNAL_HTTP_NODEPORT : If you want to use different port for HTTP Port to IO service, then export this variable with another port (Default value S3_EXTERNAL_HTTP_NODEPORT="30080").
+   8. S3_EXTERNAL_HTTPS_NODEPORT : If you want to use different port for HTTPS Port to IO service, then export this variable with another port (Default value S3_EXTERNAL_HTTPS_NODEPORT="30443").
+   9. SOLUTION_CONFIG_TYPE : There are two types config for solution.yaml file; manual and automated. 
+   - Automated means solution.yaml is created automatically but it needs VM with required specifications.
+   - Manual means we need to create solution.yaml file and replace it at script location. If you wanted to create solution yaml manually, then create it and place it at script location and configure SOLUTION_CONFIG_TYPE variable as manual (Default value SOLUTION_CONFIG_TYPE=manual).
```

Comment:
Maybe we need to rephrase this better 
Automated - solution.yaml is created by script if VM is created as per standard specification.
Manual - User needs to create solution.yaml with required disks. image details etc.